### PR TITLE
[MO] - [refactor] -> topic suite

### PR DESCRIPTION
**Pull request by see-quick**
Signed-off-by: morsak <xorsak02@stud.fit.vutbr.cz>

### Type of change

- Enhancement / new feature
- Refactoring

### Description

This PR adding a few changes to the TopicST. 

1. Removing scale test with the Kafka, where we delegate the task to the executor. Instead of using a scale, I have replaced it in just smoke test that creates & verify that topics were created.
2. Order of tests because of shared Kafka cluster, where I replaced it and with that reduce time cost.
3. Change the test when we creating topics with the Kubernetes client API and do not wait for ack and just simply verify the size of topics.
### Checklist

- [ ] Make sure all tests pass

### CODE REVIEW

**Comment by see-quick** [comment]
The aim was to execute the test where I replace the Kafka resource (config) in the last position. I have re-think my idea and do the following change.

File: systemtest/src/test/java/io/strimzi/systemtest/topic/TopicScalabilityST.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import static io.strimzi.systemtest.Constants.SCALABILITY;
+
+@Tag(SCALABILITY)
+public class TopicScalabilityST extends TopicST {
+
+    private static final Logger LOGGER = LogManager.getLogger(TopicST.class);
```
Comment (ppatierno) [inline systemtest/src/test/java/io/strimzi/systemtest/topic/TopicScalabilityST.java:19]: `TopicScalabilityST.class`

Comment (see-quick) [reply systemtest/src/test/java/io/strimzi/systemtest/topic/TopicScalabilityST.java:19]: och yeah..thanks )

**Comment by see-quick** [comment]
@strimzi-ci run tests profile=regression testcase=TopicST

**Pull request merged**